### PR TITLE
fix bug in logging username

### PR DESCRIPTION
**Pull request by youngyjd**
This PR is to pick up the change in upstream https://github.com/apache/incubator-airflow/pull/3438 and https://github.com/apache/incubator-airflow/pull/3508 for addressing the problem of bug in logging username.

### CODE REVIEW

**Comment by youngyjd** [comment]
@lyft/dp-tools-viz

File: airflow/www/templates/admin/master.html [hunk 1]
```diff
@@ -57,7 +57,11 @@
           <a class="navbar-brand" rel="home" href="{{ url_for('admin.index') }}" style="cursor: pointer;">
               <img style="float: left; width:35px; margin-top: -7px;"
                    src="{{ url_for("static", filename="pin_100.png") }}"
-                   title="{{ current_user.username }}">
+                   {% if current_user.user is defined %}
+                      title="{{ current_user.user.username }}"
+                   {% else %}
+                      title="{{ current_user.username }}"
```
Comment (ArgentFalcon) [inline airflow/www/templates/admin/master.html]: why wouldn't `current_user.user` be defined?

Comment (youngyjd) [reply airflow/www/templates/admin/master.html]: If I remember clearly, the AnonymousUserMixin() https://flask-login.readthedocs.io/en/latest/_modules/flask_login/mixins.html#AnonymousUserMixin object does not have user attribute while DefaultUser() https://github.com/apache/incubator-airflow/blob/1f0a717b65e0ea7e0127708b084baff0697f0946/airflow/default_login.py#L45 , GoogleUser() https://github.com/apache/incubator-airflow/blob/1f0a717b65e0ea7e0127708b084baff0697f0946/airflow/contrib/auth/backends/google_auth.py#L47 , etc have.

Comment (ArgentFalcon) [reply airflow/www/templates/admin/master.html]: 👍

**Comment by ArgentFalcon** [comment]
👍

**Pull request merged**